docs/source/reference/local/index.rst

```diff
@@ -11,6 +11,9 @@ SkyPilot On-prem is a lightweight cluster manager and job scheduler for local cl
 
 SkyPilot On-prem is readily deployable on top of any operating system and requires :code:`python3` to be globally installed on all machines.
 
+.. warning::
```

Comment:
Merge with master?

Btw, https://skypilot.readthedocs.io/en/latest/reference/local/index.html shows Note and Warning and they seem repetitive. Merge?